README.md

```diff
@@ -271,6 +271,13 @@ $ nix-env --install ripgrep
 $ # (Or using the attribute name, which is also ripgrep.)
 ```
 
+If you're a **Guix** user, you can install ripgrep from the official
+[package collection](https://guix.gnu.org/en/packages/ripgrep-13.0.0/):
```

Comment:
Also, sorry, I'm just noticing this now, but this URL has the version number in it. Could we replace it with a URL that doesn't have a version in it? Otherwise it's just going to bitrot. If there is no such URL, then I'd say to just remove it.

Reply:
I updated and pushed with the url removed. Thanks for the review!